migrations/V015__add_league_enable_switch.sql

```diff
@@ -0,0 +1,5 @@
+ALTER table league
+ADD COLUMN `enabled` BOOLEAN NOT NULL DEFAULT TRUE AFTER technical_name
+;
+UPDATE league SET enabled = TRUE;
+UPDATE league SET enabled = FALSE WHERE id = 4;
```

Comment:
Maybe better to put this in its own migration?

Reply:
General hint: that UPDATE can be done in one statement similar to
`UPDATE league SET enabled = (id != 4);`

Reply:
I decided to split this into two migrations. Seemed cleaner this way